ci-operator/step-registry/openshift-power-monitoring/tests/kepler/README.md

```diff
@@ -0,0 +1,27 @@
+# openshift-power-monitoring-tests-kepler
+
+<!--toc:start-->
+- [openshift-power-monitoring-tests-kepler](#openshift-power-monitoring-tests-kepler)
+  - [Purpose](#purpose)
+  - [Process](#process)
+  - [Prerequisite](#prerequisite)
+    - [Infrastructure](#infrastructure)
+<!--toc:end-->
+
+## Purpose
+
+To execute the `Power Monitoring Operator` [E2E tests](<https://github.com/openshift-power-monitoring/power-monitoring-operator/tree/v1alpha1/tests>) using the provided arguments. All the results will be copied into Artifact directory.
+
+## Process
+
+- Create a directory `/$ARTIFACT_DIR/test-run-logs` to store the test results.
+- Validate the operator installation by checking the `rollout status` and waiting for `controller` deployment to be available.
+- Execute the Operator e2e tests and capture the run inside `test-run-logs` directory. Log the events for `openshift-operators` as well as `openshift-kepler-operator` namespace while test run is executing.
```

Comment:
in the near future, the name of the namespace will also change to `openshift-power-monitoring`

Reply:
Yes will update this once we update tests as well.